scripts/copyright_check.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```

Comment:
Need a description/comment about this script at the top.

Reply:
Added description to copyright check script.